src/main/webapp/app/exam/exam-scores/exam-scores.component.ts

```diff
@@ -773,4 +716,14 @@ export class ExamScoresComponent implements OnInit, OnDestroy {
     logErrorOnSentry(errorMessage: string) {
         captureException(new Error(errorMessage));
     }
+
+    /**
+     * Formats the datalabel for every bar in order to satisfy the following pattern:
+     * number of submissions (percentage of submissions)
+     * @param value the number of submissions that fall in the grading step
+     */
+    formatDataLabel(value: any) {
```

Comment:
return is missing in the signature as well as the doc

Reply:
Done in edf3ef8